### PR TITLE
Migrate to swift 5

### PR DESCRIPTION
**Pull request by aswiftmazrana**
Making incompatible changes for Swift 5 migration.

### CODE REVIEW

**Review by traviswestbrook — APPROVED** [review]
Let me know after you get a 2nd review!

File: DrawAction/DrawText.swift [hunk 1]
```diff
@@ -49,11 +49,11 @@ final public class DrawText : DrawAction {
         paragraphStyle.lineBreakMode = lineBreakMode
         paragraphStyle.alignment = alignment
         
-        let attributes: [String : AnyObject] = [
-            NSFontAttributeName : font,
-            NSParagraphStyleAttributeName: paragraphStyle,
-            NSForegroundColorAttributeName: color,
-            NSUnderlineStyleAttributeName: underlineStyle.rawValue as AnyObject
+        let attributes: [NSAttributedString.Key : AnyObject] = [
```
Comment (skovatch) [inline DrawAction/DrawText.swift:52]: Minor, but you can get rid of the full NSAttributedString.Key prefixes. .font, .paragraphStyle, etc. will work.

File: DrawAction/DrawText.swift [hunk 2]
```diff
@@ -49,11 +49,11 @@ final public class DrawText : DrawAction {
...
+            NSAttributedString.Key.font : font,
```
Comment (skovatch) [inline DrawAction/DrawText.swift]:
```suggestion
            .font : font,
```

**Review by skovatch — APPROVED** [review]
A minor thing, but 👍 otherwise.

File: DrawAction/DrawText.swift (hunk 1 above)
Comment (aswiftmazrana) [reply DrawAction/DrawText.swift:52]: Done

**Comment by aswiftmazrana** [comment]
@traviswestbrook ready to merge

File: DrawAction/DrawText.swift (hunk 2 above)
Comment (aswiftmazrana) [reply DrawAction/DrawText.swift]: Done

**Pull request merged**